src/wrapper.js

```diff
@@ -47,8 +45,7 @@ class Wrapper {
      * @return {Promise.<string>}
      */
     retrieveFromKMS(value){
-        //Wrap in promise to catch errors calling `this.storage`
-        return this.storage.getSecret({ name: value });
+        return Promise.resolve(this.storage ? this.storage.getSecret({ name: value }) : null);
```

Comment:
This hides any failures of resolving secrets...

Reply:
I could change it so it resolves with null if this.storage isn't defined and then returns this.storage.getSecret if it is. Would that work?